EIPS/eip-x.md

```diff
@@ -30,9 +34,10 @@ This standard expands the utility of 4626 Vaults for asynchronous use cases. The
 ### Definitions:
 The existing definitions from [ERC-4626](./eip-4626.mn) apply. In addition, this spec defines:
 - request: a function call that initiates an asynchronous deposit/redemption flow
-- fulfill: the Vault's step of executing the request and enabling the user to claim the outcome of the request
-- claim: the corresponding Vault method to complete a request (e.g. `deposit` claims `shares` from `requestDeposit`)
 - pending request: the state where a request has been made but is not yet claimable
+- claimable: the Vault's step of processing the request and enabling the user to claim corresponding shares (for async deposit) or assets (for async redeem)
+- claim: the corresponding Vault method to complete a request (e.g. `deposit` claims `shares` from `requestDeposit`)
+- operator: the ethereum account specified by the requester which has has the right to claim a given request once it is claimable
```

Comment:
`requester` is not a term we otherwise use. And I think `ethereum account` is implied
```suggestion
- operator: the account specified by the sender of the request which has has the right to claim a given request once it is claimable
```